CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## vNEXT
 - Migrate unit test files to Typescript & Hardhat:
-    - ENSIntegration (#105)
+    - ENSIntegration (#105, #106)
```

Comment:
Why put this in the line of ENS integration ?

Reply:
Just because it's in the scope of the migration. I'm open for suggestions, where else do you suggest?

Reply:
Could just add on top of ENS integration "reanme testing files" (?)

Reply:
Yes [Update changelog](https://github.com/iExecBlockchainComputing/PoCo/pull/106/commits/6d09be4fca7c52f0c925c43c26ce822b9abfee96)